hearpreprocess/speech_commands.py

```diff
@@ -26,6 +26,10 @@
     "version": "v0.0.2",
     "embedding_type": "scene",
     "prediction_type": "multiclass",
+    "sample_duration": 1.0,
+    "evaluation": ["top1_acc"],
+    # The test set is 1.33 hours, so we use the entire thing
+    "max_task_duration_by_split": {"test": None},
```

Comment:
This is the key change in this PR

Reply:
Cool. Nice to have the ability to override this in the task_config